src/rule/type.js

```diff
@@ -34,12 +34,7 @@ const types = {
     }
   },
   date(value) {
-    return (
-      typeof value.getTime === 'function' &&
-      typeof value.getMonth === 'function' &&
-      typeof value.getYear === 'function' &&
-      !isNaN(value.getTime())
-    );
+    return value.toString() !== 'Invalid Date';
```

Comment:
is this  ecma standard?

Reply:
> toString()
This function returns a String value. The contents of the String are implementation-dependent, but are intended to represent the Date in the current time zone in a convenient, human-readable form.

No, it's not safe, my mistake. How about using only this line:
```
return !isNaN(value.getTime())
```

Reply:
current implementation is ok.